workflow/tests/compare.py

```diff
@@ -309,6 +306,7 @@ def remove_columns(cols):
     parser.add_argument('-x', '--export_file', help='The path of the export file.')
     parser.add_argument('-a', '--actions', action='append', choices=actions, help='The method to call.')
     args = parser.parse_args()
+    print(args)
```

Comment:
Remove this?

Reply:
We can, but I did put it there intentionally. It's convenient to see what's being called from the ci testing.